doc/source/basics.rst

```diff
@@ -0,0 +1,64 @@
+.. _qtjax_intro:
+
+************
+Jax in Qutip
+************
+
+
+.. _basic_usage:
+
+Basic Usages
+============
+
+Once imported, jax array will be available to contain ``qutip.Qobj``'s data and diffrax ODE will be available for solver (``sesolve``, ``mcsolve``, etc.).
+None of the function in the module are expected to be used directly, they will be used by qutip when the options to use this backend is used.
+
+There are many ways to tell Qutip to use the jax backend.
+
+- Pass Jax array to the Qobj constructor.
+
+.. code-block::
+
+    import qutip
+    import qutip_jax
+    import jax.numpy as jnp
+
+    jax_eye_qobj = qutip.Qobj(jnp.eye(3))
```

Comment:
Perhaps in these codeblocks we can add something at the end (`jax_eye_qobj.data` for example) to make it easier to whoever is executing this code what to expect or how to assess it jax is being used.

Reply:
Done, was it what you had in mind?

Reply:
Yes, looks great now.